### ISSUE TITLE
Height mismatch between viewing and editing template parts

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#48476]
https://user-images.githubusercontent.com/846565/221548382-0925d2d7-55df-41f5-aa0b-bea1d26a5442.mp4

The change in height as you move between browsing and editing template parts is a bit jarring. The experience would feel a bit smoother if the height was respected in the browse view as well.

**Comment by jasmussen** [comment]
Yep, this one would be very very nice to see in 6.2. @ntsekouras @youknowriad any thoughts?

**Comment by youknowriad** [comment]
I tried to solve this once, it's not as simple as it seems and may have unexpected side effects. I can give it another look tomorrow though.

**Comment by jameskoster** [comment]
I think this is fixed.
